code/game/gamemodes/clock_cult/clock_objects.dm

```diff
@@ -1362,12 +1362,6 @@
 			if(src)
 				qdel(src)
 
-/obj/effect/clockwork/spatial_gateway/Destroy()
-	visible_message("<span class='warning'>[src] flickers and closes!</span>")
-	animate(src, transform = matrix() - matrix(), time = 10)
-	spawn(10)
-		..()
```

Comment:
You should spawn something to do the animation, like an `overlay/temp` instead of removing it.